theano/gof/op.py

```diff
@@ -576,7 +576,11 @@ def _get_test_value(cls, v):
                 raise
             return ret
 
-        raise AttributeError('%s has no test value' % v)
+        sio = StringIO()
+        for subtr in getattr(v.tag, "trace", []):
+            traceback.print_list(subtr, sio)
+        detailed_err_msg = sio.getvalue()
+        raise AttributeError('%s has no test value %s' % (v, detailed_err_msg))
```

Comment:
Add a new line here for better display:

'%s has no test value %s' -> '%s has no test value \n%s'